src/Model/SetupOptions.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Platformsh\Client\Model;
+
+/**
+ * Represents Platform.sh setup options return data.
+ *
+ */
+class SetupOptions extends Resource
```

Comment:
SetupOptions is an odd "resource" name, it just generally doesn't fit the RESTful "resource"/"collection" model because it isn't a thing that I can fetch with a GET, create with a POST (containing its properties), and update with a PUT/PATCH. I wonder if it just shouldn't use Resource at all. As in we might want a SetupOptions class that actually makes the request and then wraps the returned data with appropriate methods or properties.

The other related problem here is that the class doesn't have/document any properties. Other Resource classes use this `@property-read` annotation because you can fetch from properties via __get(). But the data here is a list.

... so tl;dr I think extending Resource is getting in the way.

Reply:
I'm still learning about the API but I'm wondering if there are other API endpoints we're going to need to use in future that behave in the same way? If that's the case we probably shouldn't call it SetupOptions.